# msg/ThermalLinkFlow.msg
string node_a
string node_b
float64 conductance
float64 heat_flow
